apps/web/src/views/TradingCompetition/mobox/components/YourScore/MoboxYourScore.tsx

```diff
@@ -47,7 +50,7 @@ const MoboxYourScore: React.FC<React.PropsWithChildren<YourScoreProps>> = ({
         </RibbonWithImage>
       )}
       <ScoreCard
-        subgraphName="pancakeswap/trading-competition-v3"
+        subgraph={TC_MOBOX_SUBGRAPH}
```

Comment:
thinking maybe we should consider removing those old TC 😅

Reply:
what does TC mean actually? 😅 I'm blindly using the old name here

Reply:
oh right is it abbreviation of trading competition?

Reply:
Trading Competition lol. Just keep hosting it for claiming before